soa/tracks/ml/8.md

```diff
@@ -0,0 +1,36 @@
+# ML Track-Week 8
+
+Now, Let us boost our learning.  
+This week will introduce you to algorithms that can boost the acuuracy of your model.  
+Boosting considers many models, placed sequentially where each model tries to minimize the error obtained from the previous model.  
+
+We'll learn About  
+
+- Gradient Boosting Algorithm  
+- Extreme Boosting Algorithm  
+- Ada Boost Algorithm  
+
+
+
+### Click [here](https://github.com/kabirnagpal/SoA-ML-14/blob/master/week%208.ipynb) to view the Jupyter-Notebook.   
```

Comment:
Can we reserve headings for actual headings please?

Reply:
I didn't get this issue. Rest has been resolved!